apps/src/sites/studio/pages/levels/editors/fields/_validation_code.js

```diff
@@ -0,0 +1,21 @@
+import $ from 'jquery';
+
+$(initPage);
+
+function initPage() {
+  const freePlay = $('#level_free_play');
+  const autoValidate = $('#level_validation_enabled');
+  if (freePlay && autoValidate) {
+    freePlay.on('click', () => enforceMutualExclusion(freePlay, autoValidate));
```

Comment:
It looks like freePlay and autoValidate affect each other in the same way. Shouldn't it be one-directional?
freePlay == checked -> autoValidate checked
freePlay == unchecked -> autoValidate disabled
But checking/unchecking autoValidate shouldn't affect freePlay.

Reply:
gah, thank you! i started thinking about it as mutual exclusion and then implemented that without referring back to the actual spec. will update!

Reply:
🎉